### ISSUE TITLE
Write-Host output is sent to pipeline

### ISSUE THREAD

**Issue by ajansveld** [PaulHigin/PSThreadJob#11]
Messages originating from Write-Host or $Host.UI.WriteLine() are sent to the Output stream, which results in Receive-Job sending them to the pipeline:
```
$result = Start-ThreadJob {Write-Host foo} | Receive-Job -Wait -AutoRemoveJob
$result.Count  # Should be 0
```
This seems to originate from this code:
```
           hostUI.Output.DataAdded += (sender, dataAddedEventArgs) =>
            {
                Collection<PSObject> output = hostUI.Output.ReadAll();
                foreach (var item in output)
                {
                    _output.Add(item);
                }
            };
```
Not sure what the proper fix is, perhaps this can be removed?

**Comment by ajansveld** [comment]
FYI - I have been running with the ```_output.Add(item)``` line removed for several months; it fixes the issue and I have seen no negative side effects.

**Comment by PaulHigin** [comment]
I just wanted some way to get host output from the job.  Otherwise it is lost.  I suppose we could make it an option.

**Comment by ajansveld** [comment]
It looks like the Information stream handling was fixed in #30. Can we now remove the ```_output.Add(item)``` line (at least for ```psVersion.Major >= 5```)?

**Comment by PaulHigin** [comment]
Yeah, I think it can be removed.  It was always just a hack to try and capture host writes.

**Comment by ajansveld** [comment]
Fixed in version 2.0.0.
